src/main/java/org/variantsync/diffdetective/variation/diff/DiffTree.java

```diff
@@ -465,6 +467,38 @@ public ConsistencyResult isConsistent() {
         return ConsistencyResult.Success();
     }
 
+    public boolean isSameAs(DiffTree b) {
+        var visited = new HashSet<DiffNode>();
+        return isSameAs(this.getRoot(), b.getRoot(), visited);
+    }
+
+    private static boolean isSameAs(DiffNode a, DiffNode b, Set<DiffNode> visited) {
+        if (!visited.add(a)) {
+            return true;
+        }
+
+        if (!(
+                a.getDiffType().equals(b.getDiffType()) &&
+                a.getNodeType().equals(b.getNodeType()) &&
+                a.getFromLine().equals(b.getFromLine()) &&
+                a.getToLine().equals(b.getToLine()) &&
+                (a.getFormula() == null ? b.getFormula() == null : a.getFormula().equals(b.getFormula())) &&
+                a.getLabel().equals(b.getLabel())
```

Comment:
Oh, yes, this method is already merely identical in `DiffNode` because that change got cherry picked (with some modifications). Now just the `DiffNode` method is called: 15b616c0e1c43086d513ac597162087871e16723